benchmarks/envoy_proxy.py

```diff
@@ -92,9 +92,9 @@ def setUp(self):
                                                                port=proxy_server.server_port))
     self.proxy_server = proxy_server
 
-  def tearDown(self):
+  def tearDown(self, caplog):
```

Comment:
This adds to the benchmarking framework, specifically when Envoy proxy gets injected between the client and test server. Is the plan to add to the other fixtures in a follow up? E.g. https://github.com/envoyproxy/nighthawk/blob/master/test/integration/integration_test_fixtures.py ?

Reply:
I thought this is what this PR was doing. We are only adding that parameter to the benchmarking framework, because it inherits from the same fixtures. This PR does modify `integration_test_fixtures.py` and that is where the failure on errors / warnings happens. Is this what you meant or am I missing something?

Reply:
You are right, sorry about the noise, please disregard.